sklearn/linear_model/tests/test_ridge.py

```diff
@@ -998,16 +998,18 @@ def test_dtype_match_cholesky():
 
 @pytest.mark.parametrize(
     'solver', ['svd', 'cholesky', 'lsqr', 'sparse_cg', 'sag', 'saga'])
-def test_ridge_regression_dtype_stability(solver):
-    random_state = np.random.RandomState(0)
+@pytest.mark.parametrize('seed', range(1))
```

Comment:
This range looks a bit funny

Reply:
It's meant to make it easy to edit this test to run on 100s of different seeds. We probably need a more generic way to achieve this with a fixture.